### PR TITLE
chore(requirements): Apply correct minimal versions

### PR DESCRIPTION
**Pull request by Hipska**
The current composer.json doesn't mach the versions supported according to the documentation, this makes it hard to test the collector code for supported versions higher than PHP 7.0.

Next to that, the version checker checks for PHP 5.3+, but since only PHP 7.x+ is supported, I changed the default version value.

I hope this PR description is clear enough, the questions/steps in the request template seemed not fitting for this kind of cleanup.

### CODE REVIEW

**Comment by piRGoif** [comment]
Warning for PHPUnit the tests code isn't ready yet for versions > 6
There is a ticket somewhere in the Combodo bug tracker to update those tests.

**Comment by Hipska** [comment]
It blocks new people to be able to install/use phpunit if I recall correctly.

**Comment by Molkobain** [comment]
Yes please, remove it for now, so we can push this PR forward in the meantime.

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
    "require": {
-       "php": "7.*",
+       "php": "7.* || 8.0.* || 8.1.",
        "ext-libxml": "*"
    },
    "require-dev": {
-       "phpunit/phpunit": "^6"
+       "phpunit/phpunit": "*"
```
Comment (Hipska) [inline composer.json:7]:
```suggestion
       "phpunit/phpunit": "^6"
```

**Review by Molkobain — APPROVED** [review]

**Comment by jf-cbd** [comment]
Hello @Hipska, itop-data-collector-base extension is also compatible with PHP 8.2 and 8.3, could you please add them too so we can merge the PR ? Thanks a lot :)

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
    "require": {
-       "php": "7.*",
+       "php": "7.* || 8.0.* || 8.1.",
```
Comment (Hipska) [inline composer.json:3]:
```suggestion
       "php": "7.* || 8.* <= 8.3",
```

**Comment by jf-cbd** [comment]
Thanks for your contribution :)

**Pull request merged**